spring-kotlin/aws-ssh-server-manager/src/main/kotlin/dev/santos/awssshservermanager/dto/CreateTenantDto.kt

```diff
@@ -0,0 +1,12 @@
+package dev.santos.awssshservermanager.dto
+
+import javax.validation.constraints.Pattern
+import javax.validation.constraints.Size
+
+data class CreateTenantDto(
+  @field:Pattern(regexp = "^[a-z]+(-[a-z\\d]+)*\$")
+  @field:Size(min = 2, max = 50)
+  val name: String = "",
+  val awsApiKey: String = "",
```

Comment:
You should limit the size of each one of the fields to 250 since you allocated 250 characters in the database.